spotify_project/base/urls.py

```diff
@@ -8,4 +8,5 @@
     re_path(r'^spotify_login/$', views.spotify_login, name='login'),
     re_path(r'^callback/$', views.spotify_callback, name='callback'),
     re_path(r'^pull_data/$', views.pull_data, name='pull_data'),
+    re_path(r'^all_artists/$', views.all_artists, name='all_artists'),
```

Comment:
Since this is directly related to `artists`, it should live in the `artists` app instead of in `base`. The same will be true of the template, view, etc.